modules/prometheus-service-monitor.adoc

```diff
@@ -1,51 +0,0 @@
-// Module included in the following assemblies:
-//
-// * configuration/monitor-acs.adoc
-:_module-type: PROCEDURE
-[id="prometheus-service-monitor-example_{context}"]
-= Monitoring Central using Prometheus service monitor
-
-[role="_abstract"]
-If you are using the Prometheus Operator, you can use a service monitor to scrape the metrics from {rh-rhacs-first}.
-
-[NOTE]
-====
-If you are not using the Prometheus operator, you must edit the Prometheus configuration files to receive the data from {product-title-short}.
-====
-
-.Procedure
-. Create a new `servicemonitor.yaml` file with the following content:
-+
-[source,yaml]
-----
-apiVersion: monitoring.coreos.com/v1
```

Comment:
Didn't we agree to keep this as hint for users to set up monitoring with a custom Prometheus operator? We had customers struggling with this before.

Reply:
I will put the include in - not sure how it got deleted.

Reply:
Based on slack conversation, will delete old info about port customizing, as we have no evidence for usage and if we did, the preferred method if customers need this would be to add them to the official CRD/Helm chart.